appservice/src/SiteWrapper.ts

```diff
@@ -602,12 +612,13 @@ export class SiteWrapper {
         }
     }
 
-    private async showGitHubAuthPrompt(node: IAzureNode): Promise<void> {
-        const goToPortal: string = localize('goToPortal', 'Go to Portal');
-        const setupGithub: string = localize('GitRequired', 'Authorize Azure for GitHub under "Deployment options."');
+    private async showGitHubAuthPrompt(): Promise<void> {
+        const goToPortal: string = localize('learnMore', 'Learn More');
+        const setupGithub: string = localize('GitRequired', 'Connect your App to a GitHub repository to authorize Azure.');
```

Comment:
How about: "You must give Azure access to your Github account."

I think it sounds more like an error message and I think it makes it more obvious that you only have to do this once per account (rather than once per repository)